resources/board.py

```diff
@@ -0,0 +1,102 @@
+from flask import request, abort
+from flask_restful import Resource
+from models.board import Board as BoardModel
+
+
+class BoardResource(Resource):
+    """
+    Board 테이블의 모든 정보에 접근합니다.
+    """
+    def get(self):
+        """
+        DB 안의 모든 Board 정보들을 조회
+        :return: Board 정보들
+        """
+        board_list = BoardModel.select_board_list()
+        return board_list, 200
+
+    def post(self):
+        """
+        입력한 정보로 새 Board를 만들어 DB 안에 삽입
+        :return:
+        """
+        data = request.get_json()
```

Comment:
```suggestion
        data = request.json
```

어떤 것이 맞는 건지는 구글링 해보시면 되겠습니다. 저는 `request.json`에 익숙해서 suggestion한 것이니 취향에 맞게 하시면 됩니다